examples/embedding/embedding.cpp

```diff
@@ -61,6 +61,8 @@ int main(int argc, char ** argv) {
     }
 
     params.embedding = true;
+    // For BERT models, batch size must be equal to ubatch size
```

Comment:
```suggestion
    // For non-causal models, batch size must be equal to ubatch size
```

That's because BERT is not the only model that can be used for embeddings